iota/api.py

```diff
@@ -749,15 +751,293 @@ def were_addresses_spent_from(self, addresses):
 
         - https://docs.iota.org/docs/node-software/0.1/iri/references/api-reference#wereaddressesspentfrom
         """
-        return core.WereAddressesSpentFromCommand(self.adapter)(
+        return await core.WereAddressesSpentFromCommand(self.adapter)(
             addresses=addresses,
         )
 
+class StrictIota(AsyncStrictIota):
```

Comment:
Good idea! 
One question I have in mind is if the IDE will pick up this for code suggestions ( like intellisense)?
I really like it when I start typing a method name and it shows me the options, what arguments it takes and so on, basically the doctsrings of the method.
I don't want to loose the support for this, as it helps developers to navigate in the already complex world of iota :smiley:

Reply:
True that.  IDEs definitely won't pick up on it (I can attest to this because it was the same reason for dropping the command registry and making explicit API methods in the first place 😹).

Reply:
Well, I guess then we have to swallow the inconvenience of maintaining methods/docstrings for the convenience of users of PyOTA :)